.github/workflows/lint.yml

```diff
@@ -36,4 +36,4 @@ jobs:
       run: tox -e check-copyright
     - name: Misc checks
       run: |
-        tox -e bandit,safety
```

Comment:
If we no longer rely on safety we could also remove it from the dependencies in `pyproject.toml`.

Reply:
good idea!

Reply:
Never mind: Safety < 3.0.1 has grave issues (see https://github.com/pyupio/safety/issues/480) and the current version does not support performing a scan without logging into their service account (https://docs.safetycli.com/safety-docs/safety-cli-3/quick-start-guide):

> You will be unable to perform vulnerability scans unless you are authenticated. [Create an account and access your free trial here. ](https://platform.safetycli.com/register/)If you require assistance, please email [support@safetycli.com](mailto:support@safetycli.com).

Let's remove the dependency on Safety altogether.